### PR TITLE
Remove non-scalar aliases from already-run initial migration.

### PR DESCRIPTION
**Pull request by benwarfield-usds**
This is a terrible but harmless workaround for the liquibase bug that breaks when you use the aliasing feature of YAML too competently.

Revert this commit if and when that bug is ever fixed.

### CODE REVIEW

File: backend/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -139,12 +190,48 @@ databaseChangeLog:
...
             remarks: A provider who can (we hope) write test orders.
             columns:
-              - column: *pk_column
-              - column: *created_at_column
-              - column: *created_by_column
-              - column: *updated_at_column
-              - column: *updated_by_column
-              - column: *soft_delete_column
+              - column:
+
```
Comment (benwarfield-usds) [inline backend/src/main/resources/db/changelog/db.changelog-master.yaml:194]: Ugh.

Comment (TomNUSDS) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:194]: So. much. verbosity.

**Comment by TomNUSDS** [comment]
lgtm

**Comment by TomNUSDS** [comment]
More ugh.

**Pull request merged**